plugins/mousequery.cpp

```diff
@@ -291,10 +293,13 @@ struct mousequery_hook : public df::viewscreen_dwarfmodest
         return false;
     }
 
-    bool handleLeft(df::coord &mpos, int32_t mx, int32_t my)
+    bool handleLeft(df::coord &mpos, int32_t mx, int32_t my, int32_t depth)
     {
-        if (!(Core::getInstance().getModstate() & DFH_MOD_SHIFT))
-            mpos.z += Gui::getDepthAt(mx, my);
+        if (Core::getInstance().getModstate() & DFH_MOD_SHIFT) {
+            DEBUG(log).print("shift is down; not adjusting tile depth\n");
```

Comment:
looks like a new feature - is it left in intentionally? could be worth documenting if so.

Reply:
The feature is unchanged (though undocumented). If you hold shift while left clicking (and you're using twbt and have run multilevel >1) then you can select the tile that you're seeing below you instead of the one on the current zlevel. I could certainly document this.

Reply:
after some testing, I think it might be better to leave this feature undocumented. It works..wonkily. not always getting you to where you might expect to go.